variables.tf

```diff
@@ -55,6 +55,12 @@ variable "name" {
   description = "Used as part of the resource names to indicate they are created and used within a specific name"
 }
 
+variable "postfix" {
```

Comment:
I think that this variable name is a bit too vague for doing something so specific.